ui-icons/dist/icons.json

```diff
@@ -42,8 +42,9 @@
   "DragHandle200.svg",
   "DragHandle300.svg",
   "Gripper100.svg",
+  "LinkOut75.svg",
   "LinkOut100.svg",
   "LinkOut200.svg",
   "LinkOut300.svg",
   "LinkOut400.svg"
-]
\ No newline at end of file
+]
```

Comment:
Any objection to leaving this newline? The .vscode settings does it automatically.